src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  display: flex;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-width: 16px;
+  min-height: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url("images/star.svg");
```

Comment:
set `star.svg` to all stars, then set `star-active.svg` to only those who need it(use `nth-child`)
And combine selectors with same styles